### PR TITLE
Add rejection email for SR2024

### PR DESCRIPTION
**Pull request by sedders123**

### CODE REVIEW

File: SR2024/2023-09-27-place-rejection.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+Hello!
+
+We had a lot more interest in Student Robotics 2024 than we had places available. This has meant that unfortunately we're unable to offer you a place this year.
+
+We realise this is disappointing (we're disappointed too!), however we are limited by the number of kits we have and by the size of the venue. We are always looking for ways to support more teams and hope to be able to do so in future.
+
+If we have any early drop outs, we will be sure to let you know as you may be able to take their place, but we cannot promise anything.
+
+We hope you follow the competition on social media, and subscribe to our [mailing list](https://studentrobotics.org/compete/) to be the first to know when places open for Student Robotics 2025.
```
Comment (PeterJCLaw) [inline SR2024/2023-09-27-place-rejection.md:14]: Do we want to invite them to the competition?
Do we want to ask them to follow on social media?

**Review by PeterJCLaw — DISMISSED** [review]

File: SR2024/2023-09-27-place-rejection.md (hunk 1 above)
Comment (sedders123) [reply SR2024/2023-09-27-place-rejection.md:14]: Invite to competition: Maybe? Though I'm erring towards not.
Social Media: Feels like asking for followers/subscribers could be a tad tacky in this context

File: SR2024/2023-09-27-place-rejection.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+subject: Student Robotics 2024 is sadly full
+---
+
+Hello!
+
+We had a lot more interest in Student Robotics 2024 than we had places available. This has meant that unfortunately we're unable to offer you a place this year.
+
+We realise this is disappointing (we're disappointed too!), however we are limited by the number of kits we have and by the size of the venue. We are always looking for ways to support more teams and hope to be able to do so in future.
+
+If we have any early drop outs, we will be sure to let you know as you may be able to take their place, but we cannot promise anything.
```
Comment (Adimote) [inline SR2024/2023-09-27-place-rejection.md:12]: Explicitly mentioning the words "waiting list" would be good here.

Comment (sedders123) [reply SR2024/2023-09-27-place-rejection.md:12]: Happy with the below?
```suggestion
We've put you on our waiting list so if we have any early drop outs we will be sure to let you know. You may be able to take their place, but we cannot promise anything.
```

**Review by Adimote — APPROVED** [review]

**Pull request merged**